inference-engine/thirdparty/clDNN/kernel_selector/core/common/kernel_selector_utils.cpp

```diff
@@ -220,35 +220,44 @@ std::vector<size_t> GetOptimalLocalWorkGroupSizes(std::vector<size_t> gws, const
 bool CheckInputsOutputNoPitchSameDims(const base_params& params) {
     bool no_pitch_same_dims = true;
 
+    std::vector<std::pair <DataLayout, int>> block_layouts = {
+        {DataLayout::b_fs_yx_fsv16,          16},
+        {DataLayout::b_fs_zyx_fsv16,         16},
+        {DataLayout::b_fs_yx_fsv32,          32},
+        {DataLayout::b_fs_zyx_fsv32,         32},
+        {DataLayout::bs_fs_yx_bsv16_fsv16,   16},
+        {DataLayout::bs_fs_zyx_bsv16_fsv16,  16},
```

Comment:
Don't we need second block dim check here as well for double-blocked formats?

Reply:
I added logic for double-blocked formats.